packages/cluster/src/commands/up.js

```diff
@@ -98,13 +85,11 @@ module.exports = {
             alias: 'p',
             desc: 'Specify the port on which to expose the DHIS2 instance',
             type: 'integer',
-            default: defaults.port,
```

Comment:
If something is specified as "default" here, then it will be set in `argv`, making it hard to override when trying using `argv || cache || config || defaults` as the `argv` will always get set.

Reply:
Good idea, I'll do that.

Reply:
6979364